### DISCUSSION TITLE
How to define the most strict configuration possible for  PHP Cs fixer  ?

### DISCUSSION THREAD

**Discussion by Allan1901** [PHP-CS-Fixer/PHP-CS-Fixer#6817] [Q&A]
I’m collaborating in a opensource project :

https://bitbucket.org/AliasAPI/damnp/src/master/etc/php_cs_fixer/run_php_cs_fixer.sh

And i am finding trouble configuring PHP Cs fixer  in the most sensitive way possible, so it would detected every generally undesired line of code. If anyone has any tips, it would be very welcome.

**Comment by keradus** [comment]
what exactly you want to be sensitive about?
about style? then you need to define what style you want to have, or use one of available preset.
about strictness of code (eg `===` vs `==`)? then I suggest to base it on https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/.php-cs-fixer.dist.php#L39 , potentially with addition of https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/.php-cs-fixer.php-highest.php#L25-L26 (depending on your PHP version)

**Comment by keradus** [comment]
Hey! I'm closing this as no traffic around anymore. The idea could be valid, yet it didn't get any movement around.
If any of you see a value in it and are ready to implement it, please draft mini-RFC what you would like to achieve and how.
